### PR TITLE
Release notes for services-release 0.0.5

### PR DESCRIPTION
**Pull request by pilhuhn**
Release notes plus some more general info about hawkular-services.

### CODE REVIEW

File: src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+:jbake-tags: blog, release, hawkular-services
+
+Hawkular Services is a new base line distribution including only the basic Hawkular components, such as Alerts, Inventory and Metrics.
+The version https://repository.jboss.org/nexus/content/groups/public/org/hawkular/services/hawkular-services-dist/0.0.5.Final/hawkular-services-dist-0.0.5.Final.zip[0.0.5.Final] has just been released and is ready to be tested and integrated with other projects.
+
+== What's in this release
+
+This release has updated components:
+
+* Inventory is not at v 0.17.2, which features a new api. The old api endpoint has been moved to `/deprecated`
```
Comment (josejulio) [inline src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc]: I suppose: Inventory is _now_ at...

Comment (theute) [inline src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc]: is "now"

File: src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+* Due to the removal of Keycloak, multi tenancy is also simplified. The backend endpoints expect to receive a `Hawkular-Tenant`, so, multi tenancy is now handled on the consumer side.
+
+For more information on the details of this new distribution as well as the background, see the blog post http://www.hawkular.org/blog/2016/04/28/new-packaging.html[New Hawkular packaging]. Note that
+`hawkular-core-services` on that post is what we now know as `hawkular-services`.
+
+Hawkular Services is released every week on Tuesdays.
+
+== Get started
+To get started with Hawkular Services, https://repository.jboss.org/nexus/content/groups/public/org/hawkular/services/hawkular-services-dist/0.0.5.Final/hawkular-services-dist-0.0.5.Final.zip[download the latest release], unzip it, add a user and set the Agent to use the credentials for the user just add ed.
+It can be accomplished with the following commands:
```
Comment (theute) [inline src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc:41]: You should probably mention that Cassandra needs to be installed and running

File: src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+* Due to the changes in the authentication, Keycloak is not part of the distribution, although it can still be used via JAAS. Please refer to the Keycloak documentation on how to protect deployments.
+* Due to the removal of Keycloak, multi tenancy is also simplified. The backend endpoints expect to receive a `Hawkular-Tenant`, so, multi tenancy is now handled on the consumer side.
+
+For more information on the details of this new distribution as well as the background, see the blog post http://www.hawkular.org/blog/2016/04/28/new-packaging.html[New Hawkular packaging]. Note that
+`hawkular-core-services` on that post is what we now know as `hawkular-services`.
+
+Hawkular Services is released every week on Tuesdays.
+
+== Get started
+To get started with Hawkular Services, https://repository.jboss.org/nexus/content/groups/public/org/hawkular/services/hawkular-services-dist/0.0.5.Final/hawkular-services-dist-0.0.5.Final.zip[download the latest release], unzip it, add a user and set the Agent to use the credentials for the user just add ed.
```
Comment (josejulio) [inline src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc]: At the end of the line:
... for the user just _added_.

File: src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc (hunk 1 above)
Comment (pilhuhn) [inline src/main/jbake/content/blog/2016/07/05/hawkular-services-0.0.5.Final.adoc]: At least someone pays attention :-)))))

**Pull request merged**